tests/bootstrap.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\Dotenv\Dotenv;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+if (file_exists(dirname(__DIR__).'/.env.local')) {
+    (new Dotenv())->usePutenv(true)->bootEnv(dirname(__DIR__).'/.env.local', 'dev', []);
+}
```

Comment:
Videl jsem ze si to takle mel i v predchozich PR a evidentne to funguje, ale AFAIK spravne by melo byt proste

```suggestion
(new Dotenv())->usePutenv()->bootEnv(dirname(__DIR__).'/.env', 'dev', []);
```

* nacitat `.env` (to `.local` si to prida samo)
* `if` neni potreba, protoze `.env` vzdy existuje

Reply:
hmm, odkud jsem to vykopíroval 🤔